actions/shell_action_test.go

```diff
@@ -0,0 +1,71 @@
+// pmm-agent
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package actions
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRunShellAction(t *testing.T) {
+	// setup
+	id := "/action_id/6a479303-5081-46d0-baa0-87d6248c987b"
+	cmd := "echo"
+	p := NewShellAction(id, cmd, nil)
+	_, err := exec.LookPath(cmd)
+	if err != nil {
+		t.Skipf("Test skipped, reason: %s", err)
```

Comment:
We can rely on both `echo` and `sleep` being present. There is no need to lookup it and skip the test.